lua/cinema/construction/sh_trash.lua

```diff
@@ -90,7 +90,6 @@ TrashLocationOverrides = {
     ["SportZone"] = TRASHLOC_BUILD,
     ["SushiTheater Attic"] = TRASHLOC_BUILD,
     ["SushiTheater Basement"] = TRASHLOC_BUILD,
-    ["SushiTheater Third Floor"] = TRASHLOC_NOSPAWN,
```

Comment:
Removing the override won't allow you to build there. The default is NOSPAWN if it's a Theater or NOBUILD otherwise:
https://github.com/swampservers/contrib/blob/7e25bd39043212f480a56ab73764c823734e0828/lua/cinema/construction/entities/prop_trash/shared.lua#L298-L310

You want to set it to TRASHLOC_BUILD if you want to allow building in a location.

Reply:
No this is correct, because I want it to function like a private theater where only the owner can build and not like the auditorium where everyone can build. What you posted gets overridden later when you are the owner. (Look at line 386)
https://github.com/swampservers/contrib/blob/7e25bd39043212f480a56ab73764c823734e0828/lua/cinema/construction/entities/prop_trash/shared.lua#L365-L391

Reply:
Actually I think this is more relevant. The code above is just allowing you to tape, but this should allow the prop to exist in your theater.
https://github.com/swampservers/contrib/blob/7e25bd39043212f480a56ab73764c823734e0828/lua/cinema/construction/entities/prop_trash/shared.lua#L346-L354

Reply:
Okay, I see what you're going for. Removing that override will in fact make SushiTheater Third Floor act like a normal private theater